app/views/partials/attachments_edit_list.scala.html

```diff
@@ -32,14 +32,14 @@
 <table id="@{id}-div" aria-label="Files to attach to the ruling">
     <tr class="border-bottom row-alignment">
         <th scope="col" class="file-name-column-width bold-small w-75 pl-0">File</th>
-        <th scope="col" lass="bold-small w-25 p-0" id="header-select" >Publish</th>
+        <th scope="col" class="bold-small w-25 p-0" id="header-select" >Publish</th>
         <th scope="col" class="bold-small p-0 attachment-table-date">Date</th>
     </tr>
     @for((file: StoredAttachment, index: Int) <- attachments.zipWithIndex) {
     <tr id="@{id}-row-@{index}" class="border-bottom row-alignment">
         <td>
             <div id="@{id}-row-@{index}-title" class="file-name-column-width w-75 pl-0">
-                @attachment(s"$id-row-$index", file, useTabIndex = startAtTabIndex.map(x => x+1+(0*index)))
+                @attachment(s"$id-row-$index", file, caseRef.getOrElse(""), useTabIndex = startAtTabIndex.map(x => x+1+(0*index)))
```

Comment:
I would be really cautious about using this `getOrElse` here. I think if you do that you will want to make sure that the `caseRef` above does not have a default value to ensure that it is passed in. It might even be better to make it non-optional in the constructor for this template?